### PR TITLE
[local-path-provisioner] Move to distroless

### PR DESCRIPTION
**Pull request by name212**
## Description

local path provisioner images are now based on distroless image.

## Why do we need it, and what problem does it solve?

We'd like to base our images on a distroless image.

## Manual testing

Have two local path provisioners
image

Also have pvc and pv. Directory were created on node
image
image

Switch to pr branch
All are running
image

After add 2 sts, pvs were created successfully
image

Directories were created with right permission
image

After deleting sts and pvc one pv was kept another was delete as need
image

Directory was created too
image

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: local-path-provisioner
type: feature
summary: Move to distroless
impact_level: default
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @name212. I've started the workflow for label 'e2e/run/static'...

File: modules/031-local-path-provisioner/images/helper/dir-manager/main.go [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+func main() {
+	flag.Parse()
+	if action != SETUP && action != TEARDOWN {
+		fmt.Fprintf(os.Stderr, "Error. Incorrect action: %s", action)
+		os.Exit(1)
+	}
+
+	if path == "" {
+		os.Stderr.WriteString("Error. Path is empty")
```
Comment (RomanenkoDenys) [inline modules/031-local-path-provisioner/images/helper/dir-manager/main.go:53]: may be also use fmt.Printf ? To make the code the same.

Comment (RomanenkoDenys) [reply modules/031-local-path-provisioner/images/helper/dir-manager/main.go:53]: optional

File: modules/031-local-path-provisioner/images/helper/werf.inc.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        - '**/*'
+shell:
+  installCacheVersion: "2023-10-16.2"
+  install:
+    - export "CGO_ENABLED=0"
+    - export "GOOS=linux"
+    - export "GOARCH=amd64"
+    - cd /src
+    - echo "Build manager"
+    - go build -o /tmp/manager main.go
```
Comment (RomanenkoDenys) [inline modules/031-local-path-provisioner/images/helper/werf.inc.yaml:19]:
```suggestion
    - GOPROXY={{ $.GOPROXY }} go build -o /tmp/manager main.go
```

**Review by RomanenkoDenys — CHANGES_REQUESTED** [review]
some nits

File: modules/031-local-path-provisioner/images/helper/dir-manager/main.go (hunk 1 above)
Comment (name212) [reply modules/031-local-path-provisioner/images/helper/dir-manager/main.go]: good point. I've fixed it

**Review by RomanenkoDenys — APPROVED** [review]
LGTM

**Pull request merged**